src/index.tsx

```diff
@@ -644,613 +676,606 @@ export class App extends React.Component<any, AppState> {
+                      ...elements,
+                      {
+                        // we need to recreate the element to update dimensions &
+                        //  position
+                        ...newTextElement(element, text, element.font),
+                        isSelected: true
+                      }
+                    ];
+                  }
+                  this.setState({
+                    draggingElement: null,
+                    editingElement: null,
+                    elementType: "selection"
+                  });
                 }
-                this.setState({
-                  draggingElement: null,
-                  editingElement: null,
-                  elementType: "selection"
-                });
+              });
+            }}
+            onMouseMove={e => {
+              const hasDeselectedButton = Boolean(e.buttons);
+              if (
+                hasDeselectedButton ||
+                this.state.elementType !== "selection"
+              ) {
+                return;
               }
-            });
-          }}
-          onMouseMove={e => {
-            const hasDeselectedButton = Boolean(e.buttons);
-            if (hasDeselectedButton || this.state.elementType !== "selection") {
-              return;
-            }
-            const { x, y } = viewportCoordsToSceneCoords(e, this.state);
-            const selectedElements = elements.filter(e => e.isSelected).length;
-            if (selectedElements === 1) {
-              const resizeElement = getElementWithResizeHandler(
-                elements,
-                { x, y },
-                this.state
-              );
-              if (resizeElement && resizeElement.resizeHandle) {
-                document.documentElement.style.cursor = getCursorForResizingElement(
-                  resizeElement
+              const { x, y } = viewportCoordsToSceneCoords(e, this.state);
+              const selectedElements = elements.filter(e => e.isSelected)
+                .length;
+              if (selectedElements === 1) {
+                const resizeElement = getElementWithResizeHandler(
+                  elements,
+                  { x, y },
+                  this.state
                 );
-                return;
+                if (resizeElement && resizeElement.resizeHandle) {
+                  document.documentElement.style.cursor = getCursorForResizingElement(
+                    resizeElement
+                  );
+                  return;
+                }
               }
-            }
-            const hitElement = getElementAtPosition(elements, x, y);
-            document.documentElement.style.cursor = hitElement ? "move" : "";
-          }}
-        />
-      </div>
+              const hitElement = getElementAtPosition(elements, x, y);
+              document.documentElement.style.cursor = hitElement ? "move" : "";
+            }}
+          />
+        </div>
+        <div className="langBox">
```

Comment:
Also, the content box is too big (preventing drawing into white space around the buttons). Would fix to content as much as possible.

![image](https://user-images.githubusercontent.com/5153846/72835717-86ecad80-3c8b-11ea-8bfc-a8e88a660f65.png)